### ISSUE TITLE
[SOGo] current source cannot synchronize defaults

### ISSUE THREAD

**Issue by ghost** [mailcow/mailcow-dockerized#3659]
**Prior to placing the issue, please check following:**
- [x] I understand, that not following or deleting the below instructions, will result in immediate closing and deletion of my issue.
- [x] I have understood that answers are voluntary and community-driven, and not commercial support.
- [x] I have verified that my issue has not been already answered in the past. I also checked previous [issues](https://github.com/mailcow/mailcow-dockerized/issues).

**Description of the bug**:

Since 4.3.2.20200702-1 or maybe 4.3.2.20200628-1 nightlies we have this strange error popping on sogod logs: current source cannot synchronize defaults

So far everything is working well on our hosts but the error is indeed polluting our logging system as it happens 40 times per minute.

The bug [was reported to Inverse](https://sogo.nu/bugs/view.php?id=5070) but I believe now it's somehow related to Mailcow.

**Docker container logs of affected containers**:

```
[110]: [ERROR] <0x0x565244fb28a0[SOGoDomainDefaults]> current source cannot synchronize defaults
```

**Reproduction of said bug**:

Happens on all our hosts, they are all up-to-date.

**System information**:

| Question | Answer |
| --- | --- |
| My operating system | Debian 10 Buster |
| Is Apparmor, SELinux or similar active? | No |
| Virtualization technlogy (KVM, VMware, Xen, etc - **LXC and OpenVZ are not supported** | No |
| Server/VM specifications (Memory, CPU Cores) | 2 cores, 8GiB RAM |
| Docker Version (`docker version`) | 19.03.12 |
| Docker-Compose Version (`docker-compose version`) | 1.26.2, build eefe0d31 |
| Reverse proxy (custom solution) | No |

- Output of `git diff origin/master`, any other changes to the code? If so, **please post them**.
```
$ git diff origin/master
diff --git a/data/conf/rspamd/local.d/spamassassin.conf b/data/conf/rspamd/local.d/spamassassin.conf
deleted file mode 100644
index d091af63..00000000
--- a/data/conf/rspamd/local.d/spamassassin.conf
+++ /dev/null
@@ -1 +0,0 @@
-ruleset = "/etc/rspamd/custom/sa-rules";
```

- All third-party firewalls and custom iptables rules are unsupported. *Please check the Docker docs about how to use Docker with your own ruleset*. Nevertheless, iptabels output can help us to help you: `iptables -L -vn`, `ip6tables -L -vn`, `iptables -L -vn -t nat` and `ip6tables -L -vn -t nat`.
- DNS problems? Please run `docker exec -it $(docker ps -qf name=acme-mailcow) dig +short stackoverflow.com @172.22.1.254` (set the IP accordingly, if you changed the internal mailcow network) and post the output.

No problems on these last two points.

**Comment by andryyy** [comment]
You can provide Francis with the missing information he asked for to proceed with the issue at inverse.

**Comment by ghost** [comment]
Hi, isn't he called Francis? Or are you talking about someone else?

Anyway I saw you answered the issue over there, is it a SOGo problem then?

**Comment by andryyy** [comment]
I edited my reponse. :) 

Yes, I answered him.

**Comment by ghost** [comment]
Okay, thanks. Let’s debug this over here then. :)

**Comment by andryyy** [comment]
Fixes with dcf8274d708bd2ebd5aceae7555a6062c9cf6f41

**Comment by ghost** [comment]
> Fixes with [dcf8274](https://github.com/mailcow/mailcow-dockerized/commit/dcf8274d708bd2ebd5aceae7555a6062c9cf6f41)

Many thanks 🙏🏻

**Comment by andryyy** [comment]
Thanks for the report @ SOGo :)
